bundles/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/DSCAlarmGenericBindingProvider.java

```diff
@@ -107,13 +106,8 @@ public void processBindingConfiguration(String context, Item item, String bindin
 		
 		DSCAlarmBindingConfig config = new DSCAlarmBindingConfig(dscAlarmDeviceType, partitionId, zoneId, dscAlarmItemType);
 		addBindingConfig(item, config);
-		Set<Item> items = contextMap.get(context);
-		if (items == null) {
-			items = new HashSet<Item>();
-			contextMap.put(context, items);
-		}
-		items.add(item);
 		
+		super.processBindingConfiguration(context, item, bindingConfig);
 		logger.debug("processBindingConfiguration(): Item added: {} - {}",item.getName(),bindingConfig);
```

Comment:
Removed code duplicated what super.processBindingConfiguration had already done, so moved super call to end.